components/services/markdown/src/index.js

```diff
@@ -7,7 +7,8 @@ class ServiceMarkdown extends Component {
   state = {html: ''}
 
   async componentDidMount() {
-    const marked = await import('marked')
+    const markedLibrary = await import('marked')
+    const marked = markedLibrary.default || markedLibrary
```

Comment:
Are you sure it's not related to https://github.com/SUI-Components/sui/pull/298/files ?
Maybe it has to do with the babel preset ?

Reply:
No. We tried with phoenix with both my computer and @devtoni 's and then we edited the node_modules and it worked. And it didn't happen with sui-studio@4. Only 5.